### PR TITLE
KEN-157 when price is equal, select quote with larger qty

### PR DESCRIPTION
**Pull request by kenneththomas**

### CODE REVIEW

File: pyengine/katana.py [hunk 1]
```diff
@@ -1,12 +1,22 @@
...
+            else:
+                return False
         else:
             return False
     elif side == 'sell':
-        if p1 >= p2:
+        if p1 > p2:
             return True
+        if p1 == p2:
+            if q1 > q2:
```
Comment (TayHemmons) [inline pyengine/katana.py:16]: what if q1 == q2? question is for both buy and sell versions

File: pyengine/katana.py [hunk 2]
```diff
@@ -82,7 +93,7 @@ def quotetrimmer(side,limitprice,book):
 
     for quote in book.copy():
         quoteprice = book[quote][0]
-        if compareprice(side,limitprice,quoteprice):
+        if compareprice(side,limitprice,quoteprice,1,0):
```
Comment (TayHemmons) [inline pyengine/katana.py:96]: if you want this behavior, default q1=1 and q2=0 in the compareprice function itself dont add it here where its not needed

File: pyengine/katana.py (hunk 1 above)
Comment (kenneththomas) [reply pyengine/katana.py]: changing to >=, so we end up using the latest quote if other factors are the same

File: pyengine/katana.py (hunk 2 above)
Comment (kenneththomas) [reply pyengine/katana.py]: done

**Pull request merged**